dask/array/core.py

```diff
@@ -3217,23 +3241,18 @@ def from_array(
             else:
```

Comment:
Comment on the NumPy array inlining codepath (3226-3232):

We could easily convert this to `Blockwise` using a `BlockwiseDepDict`. Really all you'd do is remove the `[name]` from the `keys`, and `dsk` becomes the dep dict!

However, I don't actually know if there's any value in this. The amount of work, and amount of serialized data, is clearly the same. I wouldn't be surprised if blockwise and all the packing actually adds a bit of serialization overhead. I think the only benefit is that you'd get blockwise fusion.

Reply:
This was a nice idea